src/screens/addEvent/addEventStyle.ts

```diff
@@ -0,0 +1,130 @@
+import { styled } from '@mui/material';
+import { Box, Button, TextField, FormControl, MenuItem } from '@mui/material';
+
+import { theme } from '../../assets/styles/theme';
+
+export const Wrapper = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  padding: 4vh;
+  box-sizing: border-box;
+`;
+
+export const BootstrapButton = styled(Button)({
+  boxShadow: 'none',
+  textTransform: 'none',
+  fontSize: 16,
+  padding: '6px 12px',
+
+  border: '1px solid',
+  lineHeight: 1.5,
+  backgroundColor: theme.colors.red_100,
+  borderColor: theme.colors.red_100,
+  borderRadius: 8,
+  color: theme.colors.white_100,
+  margin: 20,
+  fontFamily: ['Inter'].join(','),
```

Comment:
możesz zamienić na `font-family: 'Inter', sans-serif;`

Reply:
Jak skończysz coś poprawiać / porpawiłeś już to daj Resolve conversation żeby było wiadome co i kiedy kończyłeś